src/components/ComponentPages/ProfileInfo/ProfileInfoUI/VerifyMobileNumber.tsx

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { Row, Col, Form, Input, Button } from "antd";
+import styles from "./ProfileInfo.module.scss";
+import messages from "../../../../messages";
+
+function VerifyMobileNumber() {
```

Comment:
I think the filename `VerifyMobileNumberForm` is more appropriate.  And it should be under the `Form` folder

Reply:
Done